org.ethereum.Mist.json

```diff
@@ -0,0 +1,64 @@
+{
+    "app-id": "org.ethereum.Mist",
+    "branch": "stable",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "mist",
+    "appstream-compose": true,
+    "separate-locales": false,
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
Electron doesn't support wayland.

Reply:
Huh. I didn't know that. Are they working on it?

Reply:
There is work in Chromium to get Wayland support which probably will trickle into Electron. Give it a year.